local-cli/generator-android/templates/src/app/proguard-rules.pro

```diff
@@ -40,10 +40,13 @@
 
 -keep class * extends com.facebook.react.bridge.JavaScriptModule { *; }
 -keep class * extends com.facebook.react.bridge.NativeModule { *; }
+-keep class * { native <methods>; }
```

Comment:
I tried but it didn't work. There's still errors:

```
E/dalvikvm( 1032): ERROR: couldn't find native method
E/dalvikvm( 1032): Requested: Lcom/facebook/react/bridge/ReadableNativeArray;.getType:(I)Lcom/facebook/react/bridge/ReadableType;
```

Maybe there's other problem. I'll take a look and give a update for this PR.